frontend/src/views/Evaluation/Nomination/NominationTable.tsx

```diff
@@ -16,9 +20,23 @@ interface DataTableRowProps {
 }
 
 const DataTableItemRenderer: React.FC<DataTableRowProps> = ({ item }) => (
-    <ParticipantCard participant={item.participant} />
+    <ParticipantCard key={item.participant.id} participant={item.participant} />
 )
 
+const DeleteColumn: React.FC<DataTableRowProps> = ({ item }) => {
```

Comment:
Could we call this something like (?)
```suggestion
const DeleteColumnItemRenderer: React.FC<DataTableRowProps> = ({ item }) => {
```

We should anyways call the two renderers something similar.